e2e/runner/bitcoin.go

```diff
@@ -43,6 +41,24 @@ func (runner *E2ERunner) DepositBTCWithAmount(amount float64) (txHash *chainhash
 		panic(err)
```

Comment:
Can we instead return an error and panic in the calling function?

Reply:
yeah. https://github.com/zeta-chain/node/pull/2266/commits/f40ac9277fc52f01c2145888d605296b5f89a39d